packages/protocol/test/governance/release_gold.ts

```diff
@@ -1062,86 +1068,122 @@ contract('ReleaseGold', (accounts: string[]) => {
   })
 
   describe('#expire', () => {
-    beforeEach(async () => {
-      await createNewReleaseGoldInstance(releaseGoldDefaultSchedule, web3)
-    })
-
-    describe('when called before expiration time has passed', () => {
-      it('should revert', async () => {
-        await assertRevert(releaseGoldInstance.expire({ from: releaseOwner }))
-      })
-    })
-
-    describe('when the contract has finished releasing', () => {
+    describe('when the contract is expirable', () => {
       beforeEach(async () => {
-        const timeToTravel = 12 * MONTH
-        await timeTravel(timeToTravel, web3)
+        await createNewReleaseGoldInstance(releaseGoldDefaultSchedule, web3)
       })
 
-      it('should revert before `EXPIRATION_TIME` after release schedule end', async () => {
-        await assertRevert(releaseGoldInstance.expire({ from: releaseOwner }))
+      describe('when called before expiration time has passed', () => {
+        it('should revert', async () => {
+          await assertRevert(releaseGoldInstance.expire({ from: releaseOwner }))
+        })
       })
 
-      describe('when `EXPIRATION_TIME` has passed after release schedule completion', () => {
+      describe('when the contract has finished releasing', () => {
         beforeEach(async () => {
-          const [releaseGoldStartTime, , , ,] = await releaseGoldInstance.releaseSchedule()
-          const expirationTime = await releaseGoldInstance.EXPIRATION_TIME()
-          const timeToTravel = releaseGoldStartTime.plus(expirationTime).toNumber()
+          const timeToTravel = 12 * MONTH
           await timeTravel(timeToTravel, web3)
         })
 
-        describe('when not called by releaseOwner', () => {
-          it('should revert', async () => {
-            await assertRevert(releaseGoldInstance.expire())
-          })
-        })
-
-        describe('when called by releaseOwner', () => {
-          it('should succeed', async () => {
-            await releaseGoldInstance.expire({ from: releaseOwner })
-          })
+        it('should revert before `EXPIRATION_TIME` after release schedule end', async () => {
+          await assertRevert(releaseGoldInstance.expire({ from: releaseOwner }))
         })
 
-        describe('when an instance is expired', () => {
+        describe('when `EXPIRATION_TIME` has passed after release schedule completion', () => {
           beforeEach(async () => {
-            await releaseGoldInstance.expire({ from: releaseOwner })
-          })
-
-          it('should revoke the contract', async () => {
-            const isRevoked = await releaseGoldInstance.isRevoked()
-            assert.equal(isRevoked, true)
+            const [releaseGoldStartTime, , , ,] = await releaseGoldInstance.releaseSchedule()
+            const expirationTime = await releaseGoldInstance.EXPIRATION_TIME()
+            const timeToTravel = releaseGoldStartTime.plus(expirationTime).toNumber()
```

Comment:
Is this right? I thought timetravel takes a delta, but here it seems like you're passing an absolute value

Reply:
Fixed it, you're right